pandas-stubs/core/frame.pyi

```diff
@@ -328,6 +333,46 @@ class DataFrame(NDFrame, OpsMixin):
         render_links: _bool = ...,
         encoding: _str | None = ...,
     ) -> _str: ...
+    @overload
+    def to_xml(
+        self,
+        path_or_buffer: FilePath | WriteBuffer[bytes] | WriteBuffer[str],
+        index: bool = ...,
+        root_name: str | None = ...,
+        row_name: str | None = ...,
```

Comment:
Docs don't show `None` as option for `row_name`

Reply:
Fixed.